chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+% testing randomization reproducibility
+Other commands may induce randomness in the data,
+change the sorting order,
+or alter the place of the random generator without you realizing it,
+so carefully confirm exactly how your code runs before finalizing it.\sidenote{
+	\url{https://dimewiki.worldbank.org/Randomization_in_Stata}}
+To confirm that a randomization has worked correctly before finalizing its results,
 save the outputs of the process in a temporary location,
 re-run the code, and use \texttt{cf} or \texttt{datasignature} to ensure
 nothing has changed. It is also advisable to let someone else reproduce your
 randomization results on their machine to remove any doubt that your results
 are reproducible.
+Once the result of a randomization is used in the field,
+there is no way to correct any mistakes.
 
 \codeexample{replicability.do}{./code/replicability.do}
 
 
 
-%-----------------------------------------------------------------------------------------------
-
-\section{Sampling and randomization}
-
-The sampling and randomization processes we choose
-play an important role in determining the size of the confidence intervals
-for any estimates generated from that sample,
-and therefore our ability to draw conclusions.
-Random sampling and random assignment serve different purposes.
-Random \textit{sampling} ensures that you have unbiased population estimates,
-and random \textit{assignment} ensures that you have unbiased treatment estimates.
-If you randomly sample or assign a set number of observations from a set frame,
-there are a large -- but fixed -- number of permutations which you may draw.
-
-In reality, you have to work with exactly one of them,
-so we put a lot of effort into making sure that one is a good one
-by reducing the probability of the data indicating that results or correlations are true when they are not.
-In large studies, we can use what are called \textbf{asymptotic standard errors}\sidenote{
-  \url{https://stats.stackexchange.com/q/88491}}
-to express how far away from the true population parameters our estimates are likely to be.
-These standard errors can be calculated with only two datapoints:
-the sample size and the standard deviation of the value in the chosen sample.
-They are also typically the best case scenario for the population given the data structure.
-In small studies, such as those we often see in development,
-we have to be much more careful, particularly about practical considerations
-such as determining the representative population
-and fitting any constraints on study and sample design.
-This section introduces universal basic principles for sampling and randomization.
-
-\subsection{Sampling}
+\subsection{Randomized sampling}
 
+% sampling universe: the master dataset
 \textbf{Sampling} is the process of randomly selecting units of observation
 from a master list of individuals to be included in data collection.\sidenote{
```

Comment:
master list is easily confused with master dataset. i think you do not mean the same thing. but might be better in that case to use a word other than 'master'

Reply:
I changed to "_from a list of individuals eligible to be included in data collection_" here: ae81ecc72633e01f806db3df49215ff9d4e65cb8